terraform/18f.gov.tf

```diff
@@ -1015,7 +1015,7 @@ resource "aws_route53_record" "18f_gov__acme-challenge_www_handbook_18f_gov_txt"
 
 resource "aws_route53_record" "18f_gov_www_handbook_18f_gov_a" {
```

Comment:
should the name of this resource also be changed? removing the `_` between `www` and `handbook`?

Reply:
i can change it .. i think its just a variable ...  .. i do no think that impacts dns? do yo know the impacts of changing the name?